libcxx/test/std/algorithms/alg.modifying.operations/alg.partitions/stable_partition.pass.cpp

```diff
@@ -23,38 +23,21 @@
 #include "test_iterators.h"
 #include "test_macros.h"
 
-struct is_odd
-{
-    bool operator()(const int& i) const {return i & 1;}
+struct is_odd {
+  TEST_CONSTEXPR_CXX26 bool operator()(const int& i) const { return i & 1; }
 };
```

Comment:
This is very hard to review with the formatting changes. Can you do the formatting changes in a separate patch and this have this patch on change the `constexpr` part.

Does the same applies to the other P2562 patches? Feel free to do all formatting in 1 patch and directly commit that patch.

Reply:
Done. I've also done for #129008, while I don't think #128860 needs separating formatting changes.

Reply:
Thanks! When the changes are minimal I don't mind too much. I just prefer to avoid validating whether there are only formatting changes or real changes in a patch.